ansible/build-vars.yml

```diff
@@ -0,0 +1,9 @@
+nodejs_app_name: universal
+nodejs_app_install_dir: /opt/{{ nodejs_app_name }}
```

Comment:
@gtirloni I think those are separate. Universal lives in a parent ``node_modules`` directory:
https://github.com/GPII/grunt-gpii/blob/master/tasks/gpii.js#L24-L25